components/org.wso2.carbon.identity.api.server.application.management/org.wso2.carbon.identity.api.server.application.management.common/src/main/java/org/wso2/carbon/identity/api/server/application/management/common/ApplicationManagementServiceHolder.java

```diff
@@ -85,4 +88,14 @@ public static void setoAuthServerConfiguration(OAuthServerConfiguration oAuthSer
 
         ApplicationManagementServiceHolder.oAuthServerConfiguration = oAuthServerConfiguration;
     }
+
+    public static TemplateManager getTemplateManager() {
+
+        return templateManager;
+    }
+
+    public static void setTemplateManager(TemplateManager templateManager) {
+
+        ApplicationManagementServiceHolder.templateManager = templateManager;
```

Comment:
This class is written in 2019 and this should be a singleton class...

Reply:
Fixed with a9f6e387b927fc0e84b0abc4e46a77327e60649a